spec/unit/widgets/selectionbox_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rspec'
+
+module LibyuiClient
+  module Widgets
+    RSpec.describe Selectionbox do
+      let(:widget_controller) { instance_double('widget_controller') }
+      let(:widget_filter) { FilterExtractor.new({ id: 'test_selectionbox' }) }
+      subject { Selectionbox.new(widget_controller, widget_filter) }
+      let(:item1) { 'item1' }
+      let(:item2) { 'item2' }
+      let(:item3) { 'item3' }
+      let(:items) { [item1, item2, item3] }
+      let(:response) do
+        double('Response', { body: [{ class: 'YSelectionBox',
+                                      id: 'test_selectionbox',
+                                      items: [{ label: 'item1',
+                                                selected: true },
+                                              { label: 'item2' },
+                                              { label: 'item3' }],
+                                      items_count: 3 }] })
+      end
+
+      describe '#select' do
+        it 'call select action for the defined value' do
+          allow(widget_controller).to receive(:send_action)
+          expect(subject).to receive(:action).with({ action: 'select', value: item1 })
+          expect(subject.select(item1)).to eq(subject)
```

Comment:
we could check object identity with `.to equal`

Reply:
I thought it was the same, thanks for pointing that out. I checked the differences and indeed ".to equal" is more suitable here.